packages/stockflux-components/src/components/Titlebar/Titlebar.js

```diff
@@ -43,9 +44,21 @@ export default ({ title }) => {
         <div className="header-icon" onClick={onMinimizeClick} title="Minimize">
           <MinimizeIcon />
         </div>
-        <div className="header-icon" onClick={onCloseClick} title="Close">
-          <CloseIcon />
-        </div>
+        {!confirmClose && (
+          <div className="header-icon" onClick={onCloseClick} title="Close">
+            <CloseIcon />
+          </div>
+        )}
+        {confirmClose && (
+          <div className="header-icon">
+            <ConfirmationWindow
+              message="This will close all related windows. Do you wish to continue?"
```

Comment:
Should this be `this will close the current window` instead? Or is it `all related` as in any child windows spawned specifically from this window?

Reply:
`all related` is needed as it would also close any children spawned from the window in question. 
I suspect the wording will change when UX have looked at the confirmation boxes.

Reply:
That's what I thought; thanks for the clarification.